[BUG] Debugging the Cloud Run node hello-world app doesnt work

1) run the [CR node hello-world application](https://github.com/GoogleCloudPlatform/cloud-code-samples/tree/cloud-run-updates/nodejs/nodejs-cloud-run-hello-world) using `skaffold debug`

expected: debugging works
actual: application starts up with errors:

```
Port forwarding pod/node-local-t-5f558f75b-hbmgz in namespace default, remote port 8080 -> address 127.0.0.1 port 8081
Port forwarding pod/node-local-t-5f558f75b-hbmgz in namespace default, remote port 9229 -> address 127.0.0.1 port 9229
[node-local-t-5f558f75b-hbmgz node-local-t-container] Debugger listening on ws://127.0.0.1:9229/2cae197c-b3fa-4e05-aeeb-244df89b3205
[node-local-t-5f558f75b-hbmgz node-local-t-container] For help, see: https://nodejs.org/en/docs/inspector
[node-local-t-5f558f75b-hbmgz node-local-t-container] 
[node-local-t-5f558f75b-hbmgz node-local-t-container] > nodejs-cloud-run-hello-world@0.0.1 debug /usr/src/app
[node-local-t-5f558f75b-hbmgz node-local-t-container] > node --inspect=9229 index.js
[node-local-t-5f558f75b-hbmgz node-local-t-container] 
[node-local-t-5f558f75b-hbmgz node-local-t-container] Starting inspector on 127.0.0.1:9229 failed: address already in use
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! code ELIFECYCLE
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! errno 12
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! nodejs-cloud-run-hello-world@0.0.1 debug: `node --inspect=9229 index.js`
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! Exit status 12
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! 
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! Failed at the nodejs-cloud-run-hello-world@0.0.1 debug script.
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[node-local-t-5f558f75b-hbmgz node-local-t-container] 
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR! A complete log of this run can be found in:
[node-local-t-5f558f75b-hbmgz node-local-t-container] npm ERR!     /root/.npm/_logs/2020-03-31T22_50_34_271Z-debug.log
Forwarding container node-local-t-5f558f75b-hbmgz/node-local-t-container to local port 8081.
```

(note: running with `skaffold dev` works fine)

Replacing, in the [Dockerfile](https://github.com/GoogleCloudPlatform/cloud-code-samples/blob/cloud-run-updates/nodejs/nodejs-cloud-run-hello-world/Dockerfile):
`CMD ["npm", "run", "debug"]`

With (which is how our Kubernetes samples are configured):
`ENTRYPOINT ["node", "--inspect=9229", "index.js"]`

Fixes the issue

--- a/nodejs/nodejs-cloud-run-hello-world/Dockerfile
+++ b/nodejs/nodejs-cloud-run-hello-world/Dockerfile
@@ -18,8 +18,5 @@ RUN npm install --production
 # Copy local code to the container image.
 COPY . ./
 
-# Run with debugging on container startup.
-CMD ["npm", "run", "debug"]
-
-# Comment above and uncomment below to run without debugging.
-# CMD [ "npm", "start" ]
+# Run the web service on container startup.
+CMD ["node", "index.js"]

--- a/nodejs/nodejs-cloud-run-hello-world/package.json
+++ b/nodejs/nodejs-cloud-run-hello-world/package.json
@@ -5,7 +5,6 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "debug": "node --inspect=9229 index.js",
     "lint": "eslint **.js",
     "fix": "eslint --fix *.js",
     "test": "mocha test/*.test.js --check-leaks --timeout 5000"

